SSA/Core/Util.lean

```diff
@@ -109,3 +109,45 @@ instance [inst : Cli.ParseableType τ] {n : ℕ} : Cli.ParseableType (Vector τ
       return h ▸ Vector.ofArray arr
     else
       none
+
+
+def Std.BitVec.width : Std.BitVec w → Nat := fun _ => w
+
+-- From Haskell:
+def List.toTiers : List α  → List (List α) := (List.groupBy (fun _ _ => false))
```

Comment:
I don't know the haskell equivalent, could you add a doc-string that explains what this does?

Reply:
```suggestion
-- Takes a list of values xs and transform it into tiers on which each tier is occupied by a single element from xs.
def List.toTiers : List α  → List (List α) := (List.groupBy (fun _ _ => false))
example : [1, 2, 3].toTiers = [[1], [2], [3]] := rfl
```